data/campaigns/Under_the_Burning_Suns/scenarios/11_Battle_for_Zocthanol_Isle.cfg

```diff
@@ -961,7 +958,172 @@
         [/allow_recruit]
 
         [remove_event]
-            id=jungle_warning
+            id=jungle_warning,event5v2
+        [/remove_event]
+
+        {VARIABLE keep_reached yes}
+
+        [show_objectives]
+        [/show_objectives]
+
+        # change the music playing
+        [music]
+            name=battle.ogg
+            immediate=yes
+        [/music]
+    [/event]
+
+    # Event 5 V2: Capture the orc castle, merfolk and elves return
+    # this is supposed to adress an edge case in which a player captures orc keep first
+
+    [event]
+        name=moveto
+        id=event5v2
+
+        [filter]
+            id=Kaleh
+            x=34,36,38
+            y=26,25,26
+        [/filter]
+
+        # this check feels too cautious but I would rather have it and not need it
+
+        [if]
+            [have_unit]
+                id=Zoldak
+            [/have_unit]
+
+            [then]
+                [message]
+                    speaker=Zoldak
+                    message= _ "Welcome. I already knew you were coming but I didn't expect you to arrive straight to here. In any case feel free to make use out of our camp as it seems that we have common enemies."
```

Comment:
Suggest: 'I already knew you were coming but I didn’t (including apostrophe) expect you to arrive straight **away**. In any case, (comma) feel free to make use (remove _out_) of our camp as it seems that we have common enemies.'